apps/web-app/pages/[orgSlug]/settings/org/mail/domains/[domainId].vue

```diff
@@ -534,6 +635,44 @@
                     }
                   ]" />
               </template>
+              <template #dmarc-record>
+                <SettingsDomainDnsItem
+                  text="DMARC records help receivers verify the email came from an authorized sender. If you will only be sending email via UnInbox, add a TXT record with the following value."
+                  :blocks="[
+                    {
+                      title: 'Type',
+                      value: 'TXT'
+                    },
+                    {
+                      title: 'Name',
+                      value: domainDnsQuery?.dnsRecords?.dmarc.name || '_dmarc',
+                      hasCopyButton: true
+                    },
+                    {
+                      title: 'Value',
+                      value: 'v=DMARC1; p=reject;',
```

Comment:
use dmrac record builder here or at where spf and dkim record are built.

Reply:
but this is a fixed value, never changes